lib/l10n/WalletLocalizations.dart

```diff
@@ -0,0 +1,201 @@
+          "acknowledge that the Omni Platform would not be responsible for the "
+          "negative consequences of this.",
+      'welcomePageOneButton' : 'What you need to know about your SEED',
+
+      'welcomePageTwoTitle' : 'What you need to know \nabout your SEED',
+      'welcomePageTwoContentOne' : "When registering your account, you will be asked "
+          "to save your secret phrase (SEED) and to protect your account with a password. "
+          "On normal centralized servers, special attention is paid to the password, which "
+          "can be changed and reset via email, if the need arises. However, on decentralized "
+          "platforms such as Omni, everything is arranged differently:",
+      'welcomePageTwoContentTwo' : "You use your wallet anonymously, meaning your account "
+          "is not connected to an email account or any other identifying data.",
+      'welcomePageTwoContentThree' : "Your password protects your account when working on "
+          "a certain device or browser. It is needed in order to ensure that your secret "
+          "phrase is not saved in storage.",
+      'welcomePageTwoContentFour' : "If you forget your password, you can easily create a "
+          "new one by using the account recovery form via your secret phrase. If you lose "
+          "your secret phrase, however, you will have no way to access your account.",
+      'welcomePageTwoContentFive' : "You cannot change your secret phrase. If you "
+          "accidentally sent it to someone or suspect that scammers have taken it over, "
+          "then create a new Omni wallet immediately and transfer your funds to it.",
+      'welcomePageTwoButtonBack' : 'Go Back',
+      'welcomePageTwoButtonNext' : 'Protect Yourself',
+
+      'welcomePageThreeTitle' : 'How To Protect Yourself from Phishers',
+      'welcomePageThreeContentOne' : "One of the most common forms of scamming is "
+          "phishing, which is when scammers create fake communities on Facebook or "
+          "other websites that look similar to the authentic ones.",
+      'welcomePageThreeContentTwo' : "Always check the URL: https://www.lunarx_omni.com",
+      'welcomePageThreeContentThree' : "Do not use browsers that have extensions or "
+          "plugins to access your account.",
+      'welcomePageThreeContentFour' : "Do not open emails or links from unknown senders.",
+      'welcomePageThreeContentFive' : "Regularly update your browser and operating system.",
+      'welcomePageThreeContentSix' : "Use official security software. Do not install "
+          "unknown software which could be hacked.",
+      'welcomePageThreeContentSeven' : "Do not access your wallet when using public "
+          "Wi-Fi or someone else’s device.",
+      'welcomePageThreeButtonBack' : 'Go Back',
+      'welcomePageThreeButtonNext' : 'I Understand',
+
+      'startPageAppBarTitle' : 'Omni Wallet',
```

Comment:
The format for displaying/referencing is `Omniwallet`

Reply:
Fixed.